packages/css/src/grid/grid.scss

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-grid {
+  display: grid;
+  gap: var(--amsterdam-grid-gap);
+  grid-template-columns: repeat(var(--amsterdam-grid-column-count), 1fr);
+  margin-inline: auto;
+  padding-inline: var(--amsterdam-grid-padding-inline);
+}
+
+@media screen and (min-width: $amsterdam-breakpoint-medium) {
```

Comment:
What I meant was that in other file we nest the media queries like so:

```
.amsterdam-grid {
  display: grid;
  gap: var(--amsterdam-grid-gap);
  grid-template-columns: repeat(var(--amsterdam-grid-column-count), 1fr);
  margin-inline: auto;
  padding-inline: var(--amsterdam-grid-padding-inline);

  @media screen and (min-width: $amsterdam-breakpoint-medium) {
    .amsterdam-grid {
      grid-template-columns: repeat(var(--amsterdam-grid-medium-column-count), 1fr);
    }
  }

  @media screen and (min-width: $amsterdam-breakpoint-wide) {
    .amsterdam-grid {
      grid-template-columns: repeat(var(--amsterdam-grid-wide-column-count), 1fr);
    }
  }
}
```

Avoiding nesting because it is traditional makes no sense to me, sass and even vanilla css support it and it adds to the readability and scoping of variables (if you need them changed for some reason like a user preference query).

Reply:
Thanks for clarifying, I interpreted your comment totally differently.

With ‘traditional’ I meant the `min-width: x` syntax instead of the `width > x` syntax of Media Queries Level 4. Not the nesting :)

Updated.

Reply:
yeah sorry I commented on two things.